indexer/src/main/java/au/org/aodn/esindexer/service/IndexerServiceImpl.java

```diff
@@ -216,6 +241,22 @@ protected StacCollectionModel getMappedMetadataValues(String metadataValues) thr
 
         return stacCollectionModel;
     }
+
+    private List<String> extractOrderedLabels(VocabModel vocabModel) {
+        List<String> labels = new ArrayList<>();
+
+        // Priority: DisplayLabel > AltLabels > PrefLabel
+        if (safeGet(vocabModel::getDisplayLabel).isPresent()) {
```

Comment:
you can use return List.of(....) instead of new an arraylist and if the item already a list just return it give the if and else if are exclusive.